tutorials/metrics.md

```diff
@@ -49,3 +49,126 @@ Beside the [default golang based metrics][2], CRI-O provides the following addit
 [1]: https://github.com/curl/curl
 [2]: https://github.com/prometheus/client_golang/blob/b8b56b52bdb3a79ab877c873463cadc841133360/prometheus/go_collector.go#L65-L281
 [3]: https://github.com/kubernetes/cri-api/blob/a6f63f369f6d50e9d0886f2eda63d585fbd1ab6a/pkg/apis/runtime/v1alpha2/api.proto#L34-L128
+
+## Exporting Metrics via Prometheus
+
+The CRI-O metrics exporter can be used to provide a cluster wide scraping
+endpoint for Prometheus. It is possible to either build the container image
+manually via `make metrics-exporter` or directly consume the [available image on
+quay.io][4].
+
+[4]: https://quay.io/repository/crio/metrics-exporter
```

Comment:
Not at all related to the mergeability of this PR, but there's no "Description" text on the quay.io [repo](https://quay.io/repository/crio/metrics-exporter?tab=info) and there should be.

Reply:
Yep, I added a description which links to the documentation as well.